luaa.c

```diff
@@ -1007,6 +1101,7 @@ luaA_init(xdgHandle* xdg, string_array_t *searchpath)
         { "xrdb_get_value", luaA_xrdb_get_value},
         { "kill", luaA_kill},
         { "sync", luaA_sync},
+        { "get_key_name", luaA_get_key_name},
```

Comment:
Please change this to `_get_key_name`. If you wish to expose the function, you can make a new function in `awful.keyboard` and call `_get_key_name` from there. This way people will find it faster than if it is in the `awesome` module. It also allows to break the API more easily in the future without impacting anyone.